README.md

```diff
@@ -525,6 +525,10 @@ By default, **TypeScript Node** avoids compiling files in `/node_modules/` for t
 2.  Transpiling the entire dependency tree will make your project slower
 3.  Differing behaviours between TypeScript and node.js (e.g. ES2015 modules) can result in a project that works until you decide to support a feature natively from node.js
 
+The option `--skip-ignore` or the environmental variable `TS_NODE_SKIP_IGNORE` can be used to bypass this restriction if you need to import an uncompiled TypeScript file from within a Node module. 
+
+If you need to import uncompiled TypeScript code from a Node module, you can use the command line option `--skip-ignore` or the environment variable `TS_NODE_SKIP_IGNORE` to bypass this restriction. However, if a compiled javascript file already exists, the TypeScript file will never be re-compiled regardless of these options.
```

Comment:
I'm not sure if I should include the final sentence about the compilation behavior. It _was_ surprising to me, I expected the `.ts` files to be re-compiled.

Reply:
Hmm, yeah, that relates to `preferTsExts`.  If you enable `preferTsExts` then it'll resolve to the TS file, otherwise it'll resolve to the JS file.  It has to do with the order of attempting to resolve file extensions.